.travis.yml

```
@@ -14,6 +14,18 @@ before_install:
       HOMEBREW_NO_AUTO_UPDATE=1 brew install bash grep;
       mv $(brew --prefix)/bin/ggrep $(brew --prefix)/bin/grep;
     fi
+  # Install Terraform.
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+      TERRAFORM_OS="darwin";
+    else
+      TERRAFORM_OS="$TRAVIS_OS_NAME";
+    fi
+  - curl -sLo /tmp/terraform.zip https://releases.hashicorp.com/terraform/0.11.11/terraform_0.11.11_"${TERRAFORM_OS}"_amd64.zip
+  - unzip /tmp/terraform.zip -d /tmp
+  - mkdir -p ~/bin
+  - mv /tmp/terraform ~/bin
+  - export PATH="~/bin:$PATH"
```

Use `$HOME` instead of `~` (shell magic character).

```suggestion
  - export PATH="$HOME/bin:$PATH"
```